org.scala-ide.sdt.aspects/META-INF/aop.xml

```diff
@@ -31,5 +31,6 @@
 <aspect name="scala.tools.eclipse.contribution.weaving.jdt.debug.JDIModelPresentationAspect"/>
 <aspect name="scala.tools.eclipse.contribution.weaving.pde.ui.PluginImportAspect"/>
 <aspect name="scala.tools.eclipse.contribution.weaving.jdt.launching.JUnitLaunchConfigurationTabAspect"/>
+<!-- aspect name="scala.tools.eclipse.contribution.weaving.jdt.jdi.JdiEntryGuardianAspect"/ -->
```

Comment:
uncomment it if you need to see all places where JDI is called. They are visible in scala-ide.log file